src/Stats.js

```diff
@@ -39,43 +73,72 @@ function Stats() {
 
       for (let j = 0; j < Object.keys(Log).length; j++) {
         let exercise = Log[j];
+
+        if (!['Bench', 'Squat', 'Deadlift'].includes(exercise.exerciseName)) break;
         const currentEstimatedPR = retrieveUserPR()[exercise.exerciseName].estimated1RM;
         const currentTestedPR =  retrieveUserPR()[exercise.exerciseName].tested1RM;
-        const currentEstimatedTotal = retrieveUserPR()['Total'].estimated1RM;
-        const currentTestedTotal = retrieveUserPR()['Total'].tested1RM;
 
-        if (brzyckiEquation(exercise.Weight, exercise.Reps) > currentEstimatedPR) {
-          let estimated1RM = Math.floor(brzyckiEquation(exercise.Weight, exercise.Reps));
+        if (kemmlerEquation(exercise.Weight, exercise.Reps) > currentEstimatedPR) {
+          let estimated1RM = Math.floor(kemmlerEquation(exercise.Weight, exercise.Reps));
           updateUserPR(exercise.exerciseName, estimated1RM, 'estimated1RM');
-          updateUserPR('Total', currentEstimatedTotal + estimated1RM, 'estimated1RM')
         }
 
         if (exercise.Weight > currentTestedPR) {
           updateUserPR(exercise.exerciseName, exercise.Weight, 'tested1RM')
-          updateUserPR('Total', currentTestedTotal + exercise.Weight, 'tested1RM')
         }
       }
+
+      updateUserPR('Total', retrieveUserPR()['Bench'].estimated1RM + retrieveUserPR()['Squat'].estimated1RM + retrieveUserPR()['Deadlift'].estimated1RM, 'estimated1RM')
+      updateUserPR('Total', retrieveUserPR()['Bench'].tested1RM + retrieveUserPR()['Squat'].tested1RM + retrieveUserPR()['Deadlift'].tested1RM, 'tested1RM')
     }
   }
 
+  function calculateUserExerciseLevel(exerciseName, user1RM) {
+    let levelArray = [];
+
+    for (let j = 0; j < user1RM.length; j++) {
```

Comment:
Why is 1RM an array? When I read a variable called `user1RM` I expect it to be a single value